core/src/main/java/com/googlecode/psiprobe/AbstractTomcatContainer.java

```diff
@@ -68,6 +69,9 @@ public boolean installContext(String contextName) throws Exception {
     return findContext(contextName) != null;
   }
 
+  /* (non-Javadoc)
```

Comment:
Did at least the bulk of issues clear up?


Sent by Outlook for Android





On Wed, Oct 28, 2015 at 9:41 PM -0700, "Mark Lewis" <notifications@github.com> wrote:
> @@ -68,6 +69,9 @@ public boolean installContext(String contextName) throws Exception {
>      return findContext(contextName) != null;
>    }
>
> +  /* (non-Javadoc)

The answer is no. :)

---
Reply to this email directly or view it on GitHub:
https://github.com/psi-probe/psi-probe/pull/555/files#r43348531